src/security/decorators.py

```diff
@@ -697,13 +866,22 @@ def wrapper(request, *args, **kwargs):
         typeset_id = kwargs.get('typeset_id', None)
 
         if not base_check(request):
-            return redirect('{0}?next={1}'.format(reverse('core_login'), request.path_info))
+            return redirect(
+                '{0}?next={1}'.format(
+                    reverse('core_login'),
+                    request.path_info
+                )
+            )
 
         if request.user.is_editor(request) or request.user.is_staff:
             return func(request, *args, **kwargs)
 
-        typeset_assignment = get_object_or_404(production_models.TypesetTask, pk=typeset_id)
-        if request.user == typeset_assignment.typesetter and not typeset_assignment.completed and \
+        typeset_assignment = get_object_or_404(
+            production_models.TypesetTask,
+            pk=typeset_id
+        )
+        if request.user == typeset_assignment.typesetter and\
+                not typeset_assignment.completed and \
```

Comment:
Backslashes are evil and operator should be at the beginning of the line, is autopep8 doing this?

I think we should leave this kind of task for its own issue instead.

Reply:
Dropped autopep8 crap from this module.